### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by timknight** [Code-Pop/vue-mastery#19]
I wanted to get a list started with various accessibility issues I’ve found. I think general accessibility problem can lead to issues in user experience for all users.

* Header logo doesn’t have alternate text
* Free lessons thumbnails and course badges don’t have alternative text. If you feel this are merely for decor the set a blank alt attribute, otherwise, without an alt attribute, the full image path would be read by assistive technologies.
* Footer logo doesn’t have alternate text.
* The timestamps with lessons currently use a label tag which would be classified as an “orphaned label” since they aren’t actually labeling an element which might create confusion in assistive technology. Instead I’d suggest a pattern like so: `<span aria-label="3 minutes, 21 seconds">3:21</span>`
* The more buttons used on the homepage to take people to the classes should have added context even if hidden only for assistive technology. Using “More Classes” or “More Lessons” and then hiding the noun to only assistive technology would help provide context. You already have a class for this in your CSS… `<a href="/courses" class="button primary border">More <span class="visually-hidden">Classes</span></a>`
* Social icons in the footer have no alternate text. Consider adding real text next to the icon and then hiding it only for assistive technologies. Use your `visually-hidden` class like above.
* Your contact form uses visually hidden labels which is awesome, but those labels aren’t associated with their related fields using `for`.
* On your contact page, make sure you add alternate text for those social buttons as well.
* For the form action button, good UX would suggest that you use something my contextually relevant than just “Submit”. Perhaps “Send My Message”, “Send My Note”, “Send”? Whatever you feel is more in line with who you are.
* The Let's Get You Signed Up modal appears with an, "This email is invalid" error message immediately before interacting with the form. This can make the prospective user feel like they've done something wrong. Instead just set the focus to the email address field.

**Comment by pieer** [comment]
Thanks for all your valuable feedbacks Tim!
